File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        
        right = n
        res = list()
        for left in range(n):
            res.append(nums[left])
            res.append(nums[left+right])
        return res

File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] result = new int[2 * n];

        for (int i = 0; i < n; i++) {
            // Place element from first half
            result[2 * i] = nums[i];
            // Place element from second half
            result[2 * i + 1] = nums[n + i];
        }

        return result;
    }
}